FirebaseCore/Tests/Unit/FIRHeartbeatInfoTest.m

```diff
@@ -29,7 +29,13 @@ @implementation FIRHeartbeatInfoTest
 - (void)setUp {
   NSString *const kHeartbeatStorageFile = @"HEARTBEAT_INFO_STORAGE";
   self.dataStorage = [[GULHeartbeatDateStorage alloc] initWithFileName:kHeartbeatStorageFile];
-  NSDate *pastTime = [NSDate dateWithTimeIntervalSinceNow:-996400];
+  NSDateComponents *componentsToAdd = [[NSDateComponents alloc] init];
+  componentsToAdd.day = -1;
+
+  NSDate *pastTime = [[NSCalendar currentCalendar] dateByAddingComponents:componentsToAdd
```

Comment:
nit: maybe rename `pastTime` to smt like `dayAgo` or `yesterday`.  It looks a bit nicer to me, but I'm fine if you leave it as is.